content/go/concepts/interfaces/interfaces.md

```diff
@@ -0,0 +1,271 @@
+---
+Title: 'Interfaces'
+Description: 'An interface is a set of method signatures.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Control Flow'
+  - 'Interfaces'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+<h2>Creating an Interface</h2>
+
+An interface is composed of set of method signatures. These method signatures define the input and return values of which a data type or struct can conform to. In order to implement an interface, the type or struct must satisfy all the defined method signatures.
```

Comment:
Here, I do mean structure as in [data structure](https://www.codecademy.com/resources/docs/general/data-structures), not `struct`.